src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -61,6 +62,8 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
 }) => {
     const [value, setValue] = useState<Descendant[]>(() => parseRawValue(initialValue));
     const debouncedValue = useDebounce(value, ON_SAVE_DELAY_IN_MS);
+    const wrapperRef = createRef<HTMLDivElement>();
+    const [placeholderElement, setPlaceholderElement] = useState<Element | null>();
```

Comment:
you could set the wrapperStyle in the state directly, so you don't have to define a `placeholderElement`

Reply:
Done